docs/Testing.rst

```diff
@@ -91,6 +95,10 @@ Besides ``check-swift``, other targets are also available. Here's the full list:
 
   Runs all tests (primary, validation, and long).
 
+* ``SwiftUnitTests``
+
+  Runs all unit tests.
```

Comment:
I'm pretty sure this only *builds* the unit tests. They still have to be invoked, either individually or through lit.